src/Skylelo/BetterCosmetics/PlayerJoinEvent.php

```diff
@@ -26,4 +26,13 @@ public function onPlayerInteraction(PlayerInteractEvent $event)
             $this->plugin->particlesUI($player);
         }
     }
-}
\ No newline at end of file
+
+    public function onJoin(\pocketmine\event\player\PlayerJoinEvent $event) {
```

Comment:
Why is this `\pocketmine\event\player\PlayerJoinEvent`?
Instead of `PlayerJoinEvent` and then `use pocketmine\event\player\PlayerJoinEvent`?

The curly brace should be on a new line (if you look at the function above)